tests/nano_arena.rs

```diff
@@ -0,0 +1,22 @@
+use compact_arena::in_nano_arena;
+
+#[test]
+fn add_255_objects() {
+    in_nano_arena!(arena, {
+        for _ in 0..255 {
```

Comment:
I changed NanoArena to allow for 256 elements. Also we can now use `mk_nano_arena(arena);` without requiring a new scope.